hooks/run_mypy.sh

```diff
@@ -4,7 +4,7 @@
 set -o errexit
 
 scriptDir=$(realpath ${BASH_SOURCE[0]} --canonicalize-existing)
```

Comment:
what's your opinion on naming conventions? I think in bash it's common to use snake_case variable names or CONSTANT_CASE for environment stuff

Reply:
I am a big proponent. You are right, I just checked, I will fix the names. Should have done that in the beginning 🤔